app/src/main/res/drawable-anydpi/ic_search_blue_24dp.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/menu_item_tint"
```

Comment:
No, I didn't meant using attr here. I wanted it on the line that I reviewed previously. This is a release blocker issue, so a little urgent, sorry for the rush. It will be quicker if I send the PR. Let me send the PR and close this one. Then you can see what I meant.